OUDS/Core/OUDS/Sources/_OUDS.docc/Tokens.md

```diff
@@ -98,7 +98,7 @@ In addition, there are hundreds of _semantics tokens_ and we needed to add them
 
 That is the reason why tokens are exposed as `@objc open` to be available and overridable anywhere. 
 
-To keep the same semantics as the ones used in our specifications, _typealias_ are used to as to make the links to _primitive types_ and our logic of _tokens_. These type aliases are avaialble for those who want too make their own theme. It's only syntaxic sugar to bring _design words_ in our product.
+To keep the same semantics as the ones used in our specifications, _typealias_ are used to as to make the links to _primitive types_ and our logic of _tokens_. These type aliases are available for those who want too make their own theme. It's only syntaxic sugar to bring _design words_ in our product.
```

Comment:
Il y a une petite erreur de frappe : 'too' devrait être corrigé en 'to'.

### Correction :
_These type aliases are available for those who want **too** make their own theme_

Reply:
Well spotted!